app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/MosaicService.scala

```diff
@@ -42,190 +39,185 @@ class MosaicService[LayerStore: ProjectStore, HistStore, ToolStore](
   val authorizers = new Authorizers(xa)
 
   val routes: AuthedService[User, IO] =
-    H.handle {
-      ForeignError.handle {
-        AuthedService {
-          case GET -> Root / UUIDWrapper(projectId) / "layers" / UUIDWrapper(
-                layerId) / IntVar(z) / IntVar(x) / IntVar(y) :? BandOverrideQueryParamDecoder(
-                bandOverride) as user =>
-            val polygonBbox: Projected[Polygon] =
-              TileUtils.getTileBounds(z, x, y)
-            val eval = LayerTms.identity(
-              layers.read(layerId, Some(polygonBbox), bandOverride, None)
-            )
-
-            for {
-              fiberAuthProject <- authorizers.authProject(user, projectId).start
-              fiberAuthLayer <- authorizers
-                .authProjectLayer(projectId, layerId)
-                .start
-              fiberResp <- eval(z, x, y).start
-              _ <- (fiberAuthProject, fiberAuthLayer).tupled.join
-                .handleErrorWith { error =>
-                  fiberResp.cancel *> IO.raiseError(error)
+    AuthedService {
```

Comment:
These big sections are indentation changes, since error handling was moved out to `Main.scala`